services/blockchain-app-registry/shared/constants.js

```diff
@@ -19,7 +19,10 @@ const KV_STORE_KEY = {
 	COMMIT_HASH_UNTIL_LAST_SYNC: 'commitHashUntilLastSync',
 };
 
+const META_FILES = ['app.json', 'nativetokens.json', '.png', '.svg'];
```

Comment:
Maybe split them and rename the resulting constants to `ALLOWED_FILES` and `ALLOWED_FILE_EXTENSIONS`.